src/meta/src/lib.rs

```diff
@@ -192,6 +192,9 @@ pub fn start(opts: MetaNodeOpts) -> Pin<Box<dyn Future<Output = ()> + Send>> {
         .await
         .unwrap();
         join_handle.await.unwrap();
-        tracing::info!("Meta server is stopped");
+        match shutdown_send.send(()) {
```

Comment:
The `shutdown_send` should be called by internal logic such as a follower becoming a leader. That doesn't make sense to call send here, since the rpc service is already stopped. So we can just ignore `shutdown_send` for now and use it when we need for HA.

Reply:
Makes total sense. Will remove it